### DISCUSSION TITLE
[Node Operator Question] <For op and base network is it okay to use v1.101411.0-rc.1 tag release for op geth?>

### DISCUSSION THREAD

**Discussion by ScriptLike** [ethereum-optimism/developers#617] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

Hi,

The latest release version of op geth is from august with tag v1.101408.0. Theres some new geth features that I would like to use for optimism and base. Would it be okay/safe to use [v1.101411.0-rc.1](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101411.0-rc.1) for both chains?

Would also like to know what rc.1 stands for if possible thanks.

### Steps to Reproduce

n/a

### Which component is affected?

- [X] op-geth
- [ ] op-node
- [ ] Other

### Expected vs. Actual Behavior

n/a

### Environment Details

Ubuntu

### Protocol Information

[v1.101411.0-rc.1](https://github.com/ethereum-optimism/op-geth/releases/tag/v1.101411.0-rc.1)

**Comment by imtipi** [comment]
not recommend if you want to use this in produce environment.because it has potential bugs

> Would also like to know what rc.1 stands for if possible thanks.

this is the result from google

> RC - Release Candidate. An RC version usually includes fixes for the bugs found during the Beta phase.
